tensorflow/compiler/aot/flags.cc

```diff
@@ -24,6 +24,8 @@ void AppendMainFlags(std::vector<Flag>* flag_list, MainFlags* flags) {
        "Input GraphDef file.  If the file ends in '.pbtxt' it is expected to "
        "be in the human-readable proto text format, otherwise it is expected "
        "to be in the proto binary format."},
+      {"hlo_module", &flags->hlo_module,
```

Comment:
State that the user may supply one of `--graph` or `--hlo_module`.

Reply:
Please also add a warning that the HLO proto is *not* a backwards / forwards compatible format and it is not okay to e.g. expect an HLO module generated today will continue to work in perpetuity with tfcompile.